src/test/dbwrapper_tests.cpp

```diff
@@ -24,8 +24,8 @@ BOOST_FIXTURE_TEST_SUITE(dbwrapper_tests, BasicTestingSetup)
 BOOST_AUTO_TEST_CASE(dbwrapper)
 {
     // Perform tests both obfuscated and non-obfuscated.
-    for (int i = 0; i < 2; i++) {
-        bool obfuscate = (bool)i;
+    for (bool obfuscate: { false, true } )
```

Comment:
Nit: Please use the form `for (int x: {1, 2})` instead of `for (int x: { 1, 2 } )`.

The former is the dominant form used in the repo:

``` shell
$ git grep -E "\{(true|false|0|1|2)" -- "*.cpp" | wc -l
112
$ git grep -E "\{ (true|false|0|1|2)" -- "*.cpp" | wc -l
9
```